### PR TITLE
Feature/parmlib

### PR DESCRIPTION
**Pull request by Martin-Zeithaml**
### Description
There is a possibility to use combination of `FILE` or `PARMLIB` when specifying configuration list for `zwe` command or launcher.

This feature was probably added in zowe 2.3.0 (I am not sure, but it is there for a while).

### CODE REVIEW

File: docs/user-guide/configmgr-using.md [hunk 1]
```diff
@@ -95,12 +95,18 @@ From the GitHub links above, if you want to see changes between versions, you ca
...
 
-When using a single unix file, the syntax is just the path to the file, such as `CONFIG=/my/zowe.yaml`. However, when using multiple files, you must use the syntax `FILE(file1):FILE(file2):...` where each file is surrounded with `FILE()` and files are separated by the colon `:` character. The use of `FILE()` will allow Zowe to support other types of storage in the future. An example of using multiple configuration files would be as follows: 
+When using a single unix file, the syntax is just the path to the file, such as `CONFIG=/my/zowe.yaml`. However, when using multiple files, you must use the syntax `FILE(file1):PARMLIB(DSN(MEMBER)):...` where each file is surrounded with `FILE()` or `PARMLIB()` and files are separated by the colon `:` character. An example of using multiple configuration files would be as follows:  
 
 ```
-CONFIG=FILE(/home/me/zowe-customizations.yaml):FILE(/global/zowe/example-zowe.yaml)
+CONFIG=FILE(/home/me/zowe-customizations.yaml):FILE(/global/zowe/example-zowe.yaml):PARMLIB(MYORG.ZOWE.PARMLIB(YAML))
 ```
 
+**Note:** All `PARMLIB()` entries must all have the same member name:
```
Comment (anaxceron) [inline docs/user-guide/configmgr-using.md:104]: Please remove one of the two uses of "all" -- such as: All PARMLIB() entries must have the same member name

File: docs/user-guide/configmgr-using.md [hunk 2]
```diff
@@ -95,12 +95,18 @@ From the GitHub links above, if you want to see changes between versions, you ca
 When `zwe` is using Configuration Manager, the `CONFIG=` parameter in the z/OS ZWESLSTC JCL and the `--config` parameter in any `zwe` command that supports `--configmgr` can take a list of YAML file locations as an alternative to the backward-compatible single YAML file used in prior Zowe versions.
 
 
-When using a single unix file, the syntax is just the path to the file, such as `CONFIG=/my/zowe.yaml`. However, when using multiple files, you must use the syntax `FILE(file1):FILE(file2):...` where each file is surrounded with `FILE()` and files are separated by the colon `:` character. The use of `FILE()` will allow Zowe to support other types of storage in the future. An example of using multiple configuration files would be as follows: 
+When using a single unix file, the syntax is just the path to the file, such as `CONFIG=/my/zowe.yaml`. However, when using multiple files, you must use the syntax `FILE(file1):PARMLIB(DSN(MEMBER)):...` where each file is surrounded with `FILE()` or `PARMLIB()` and files are separated by the colon `:` character. An example of using multiple configuration files would be as follows:
```
Comment (anaxceron) [inline docs/user-guide/configmgr-using.md:98]: Isn't Unix capitalized?

Comment (anaxceron) [inline docs/user-guide/configmgr-using.md:98]: re: "when using multiple files" -- should we specify the types of files here? "when using multiple Unix files" or "when using multiple types of files"?

Comment (Martin-Zeithaml) [reply docs/user-guide/configmgr-using.md:98]: This is something I was considering. From the technical point of view, you can use Unix file or dataset with member. In the deleted sentence it is called as "different data storage types".  This phrase was too long for me, that's why I have kept only file to refer file or dataset.

Comment (anaxceron) [reply docs/user-guide/configmgr-using.md:98]: OK. Please capitalize and I can approve

File: docs/user-guide/configmgr-using.md (hunk 1 above)
Comment (anaxceron) [reply docs/user-guide/configmgr-using.md:104]: Once you remove, I can approve the PR

File: docs/user-guide/configmgr-using.md (hunk 2 above)
Comment (Martin-Zeithaml) [reply docs/user-guide/configmgr-using.md]: Unix capitalized.

Comment (Martin-Zeithaml) [reply docs/user-guide/configmgr-using.md]: I have reconsider it and change it to storage type.

**Review by anaxceron — APPROVED** [review]
Looks good! Thanks for updating the doc :grin:

**Pull request merged**